### PR TITLE
[PPML]Update readthedoc of PPML

### PR DESCRIPTION
**Pull request by Forest216**

### CODE REVIEW

File: docs/readthedocs/source/doc/PPML/Overview/ppml.md [hunk 1]
```diff
@@ -222,39 +222,177 @@ The result should look like this:
 
 #### 2.2.3 Run Trusted Big Data and ML on Cluster
 
-##### 2.2.3.1 Configure the Environment
+WARNING: If you want spark standalone mode, please refer to [standalone/README.md](https://github.com/intel-analytics/BigDL/blob/main/ppml/trusted-big-data-ml/python/docker-graphene/standalone/README.md). But it is not recommended.
 
-Prerequisite: [ssh login without password](http://www.linuxproblem.org/art_9.html) for all the nodes.
+Follow the guide below to run Spark on Kubernetes manually. Alternatively, you can also use Helm to set everything up automatically. See [kubernetes/README.md](https://github.com/intel-analytics/BigDL/blob/main/ppml/trusted-big-data-ml/python/docker-graphene/kubernetes/README.md).
+
+##### 2.2.3.1 Prepare data, key and password
```
Comment (qiyuangong) [inline docs/readthedocs/source/doc/PPML/Overview/ppml.md:229]: Merge 2.2.3.1 Prepare data & 2.2.3. 2K8S configurations into 2.2.3.1 Configure the Environment

File: docs/readthedocs/source/doc/PPML/Overview/ppml.md [hunk 2]
```diff
@@ -222,39 +222,177 @@ The result should look like this:
...
+kubectl apply -f keys/keys.yaml
+kubectl apply -f password/password.yaml
+cd kubernetes
+bash enclave-key-to-secret.sh
 ```
-##### 2.2.3.2 Start Distributed Big Data and ML Platform
+##### 2.2.3.2 Prepare the k8s configurations
 
-Firstly run the following command to start the service:
+Create the RBAC:
```
Comment (qiyuangong) [inline docs/readthedocs/source/doc/PPML/Overview/ppml.md:241]: RBAC (Role-based access control) and add link (https://kubernetes.io/docs/reference/access-authn-authz/rbac/)

File: docs/readthedocs/source/doc/PPML/Overview/ppml.md [hunk 3]
```diff
@@ -222,39 +222,177 @@ The result should look like this:
...
+    -e SGX_ENABLED=true \
+    -e SGX_LOG_LEVEL=error \
+    -e SPARK_MODE=client \
+    -e LOCAL_IP=$LOCAL_IP \
+    $DOCKER_IMAGE bash
+```
+
+##### 2.2.3.4 Init the client and run Spark applications on k8s
+
+##### 2.2.3.4.1 Init the Spark local k8s client
```
Comment (qiyuangong) [inline docs/readthedocs/source/doc/PPML/Overview/ppml.md:312]: 2.2.3.4.1 -> 1. Just like https://bigdl.readthedocs.io/en/latest/doc/PPML/Overview/ppml.html#id3

File: docs/readthedocs/source/doc/PPML/Overview/ppml.md [hunk 4]
```diff
@@ -222,39 +222,177 @@ The result should look like this:
...
+
+First, run `docker exec -it spark-local-k8s-client bash` to entry the container. Then run the following command to init the Spark local k8s client
+
+```bash
+./init.sh
+```
+
+##### 2.2.3.4.2 Configure spark-executor-template.yaml in the container
```
Comment (qiyuangong) [inline docs/readthedocs/source/doc/PPML/Overview/ppml.md:320]: 2.2.3.4.2 -> 2

File: docs/readthedocs/source/doc/PPML/Overview/ppml.md [hunk 5]
```diff
@@ -222,39 +222,177 @@ The result should look like this:
...
+
+We assume you have a working Network File System (NFS) configured for your Kubernetes cluster. Configure the `nfsvolumeclaim` on the last line to the name of the Persistent Volume Claim (PVC) of your NFS.
+
+Please prepare the following and put them in your NFS directory:
+
+- The data (in a directory called `data`)
+- The kubeconfig file.
+
+##### 2.2.3.4.3 Spark-Pi example
```
Comment (qiyuangong) [inline docs/readthedocs/source/doc/PPML/Overview/ppml.md:329]: 2.2.3.4.3 -> 3

File: docs/readthedocs/source/doc/PPML/Overview/ppml.md [hunk 6]
```diff
@@ -222,39 +222,177 @@ The result should look like this:
...
+##### 2.2.3.3  Start the client container
+
+Configure the environment variables in the following script before running it. Check [Bigdl ppml SGX related configurations](https://github.com/intel-analytics/BigDL/tree/main/ppml/trusted-big-data-ml/python/docker-graphene#1-bigdl-ppml-sgx-related-configurations) for detailed memory configurations. 
 
 ```bash
-./undeploy-distributed-standalone-spark.sh
-```
+export K8S_MASTER=k8s://$( sudo kubectl cluster-info | grep 'https.*' -o -m 1 )
+echo The k8s master is $K8S_MASTER .
+export ENCLAVE_KEY=/YOUR_DIR/enclave-key.pem
```
Comment (qiyuangong) [inline docs/readthedocs/source/doc/PPML/Overview/ppml.md:266]: remind user to modify your_xxxx

File: docs/readthedocs/source/doc/PPML/Overview/ppml.md [hunk 7]
```diff
@@ -222,39 +222,177 @@ The result should look like this:
...
+    --conf spark.ssl.keyStorePassword=$secure_password \
+    --conf spark.ssl.keyStoreType=JKS \
+    --conf spark.ssl.trustStore=/ppml/trusted-big-data-ml/work/keys/keystore.jks \
+    --conf spark.ssl.trustStorePassword=$secure_password \
+    --conf spark.ssl.trustStoreType=JKS \
+    --class org.apache.spark.examples.SparkPi \
+    --verbose \
+    local:///ppml/trusted-big-data-ml/work/spark-3.1.2/examples/jars/spark-examples_2.12-3.1.2.jar 100 2>&1 | tee spark-pi-sgx-$SPARK_MODE.log
+```
+
```
Comment (qiyuangong) [inline docs/readthedocs/source/doc/PPML/Overview/ppml.md:391]: We can add a string output (`kubectl get po`). to Tell user, what they can get.
Tell them, they can get output with `kubectl logs -f driver-xxxx`

File: docs/readthedocs/source/doc/PPML/Overview/ppml.md (hunk 6 above)
Comment (Forest216) [reply docs/readthedocs/source/doc/PPML/Overview/ppml.md:264]: update in https://github.com/intel-analytics/BigDL/pull/4515/files#diff-206dc464459fee64731e43f5909c3e6144c71a8f9711eec547deb6ccf7e404e7R246 and https://github.com/intel-analytics/BigDL/pull/4515/files#diff-206dc464459fee64731e43f5909c3e6144c71a8f9711eec547deb6ccf7e404e7R259

File: docs/readthedocs/source/doc/PPML/Overview/ppml.md (hunk 7 above)
Comment (Forest216) [reply docs/readthedocs/source/doc/PPML/Overview/ppml.md:381]: update in https://github.com/intel-analytics/BigDL/pull/4515/files#diff-206dc464459fee64731e43f5909c3e6144c71a8f9711eec547deb6ccf7e404e7R321

File: docs/readthedocs/source/doc/PPML/Overview/ppml.md (hunk 2 above)
Comment (Forest216) [reply docs/readthedocs/source/doc/PPML/Overview/ppml.md]: update in https://github.com/intel-analytics/BigDL/pull/4515/files#diff-206dc464459fee64731e43f5909c3e6144c71a8f9711eec547deb6ccf7e404e7R239

File: docs/readthedocs/source/doc/PPML/Overview/ppml.md (hunk 1 above)
Comment (Forest216) [reply docs/readthedocs/source/doc/PPML/Overview/ppml.md]: update in https://github.com/intel-analytics/BigDL/pull/4515/files#diff-206dc464459fee64731e43f5909c3e6144c71a8f9711eec547deb6ccf7e404e7R229

File: docs/readthedocs/source/doc/PPML/Overview/ppml.md (hunk 3 above)
Comment (Forest216) [reply docs/readthedocs/source/doc/PPML/Overview/ppml.md]: update in https://github.com/intel-analytics/BigDL/pull/4515/files#diff-206dc464459fee64731e43f5909c3e6144c71a8f9711eec547deb6ccf7e404e7R310

File: docs/readthedocs/source/doc/PPML/Overview/ppml.md (hunk 4 above)
Comment (Forest216) [reply docs/readthedocs/source/doc/PPML/Overview/ppml.md]: update in https://github.com/intel-analytics/BigDL/pull/4515/files#diff-206dc464459fee64731e43f5909c3e6144c71a8f9711eec547deb6ccf7e404e7R316

File: docs/readthedocs/source/doc/PPML/Overview/ppml.md (hunk 5 above)
Comment (Forest216) [reply docs/readthedocs/source/doc/PPML/Overview/ppml.md]: update in https://github.com/intel-analytics/BigDL/pull/4515/files#diff-206dc464459fee64731e43f5909c3e6144c71a8f9711eec547deb6ccf7e404e7R321

**Review by qiyuangong — APPROVED** [review]
LGTM

**Pull request merged**